public/src/store.jsx

```diff
@@ -25,6 +26,8 @@ const rootReducer = (state, action) => {
 
 const middleware = [immutable(), createLogger(), promise(), thunk];
 
-const store = createStore(rootReducer, compose(applyMiddleware(...middleware)));
+const store = createStore(rootReducer, compose(applyMiddleware(...middleware), autoRehydrate()));
+
+persistStore(store);
```

Comment:
@andipants12 How often is persistStore() going to be called?
Essentially, how often is the new store going to be persisted?